lib/widgets/app.dart

```diff
@@ -149,12 +149,23 @@ class MessageListPage extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(title: const Text("All messages")),
-        body: Center(
-            child: Column(children: const [
-          Expanded(child: MessageList()),
-          SizedBox(
-              height: 80,
-              child: Center(child: Text("(Compose box goes here.)"))),
-        ])));
+
+        // See MediaQuery.removePadding for why we need this BuildContext.
```

Comment:
This comment can be dropped; when a reader wants to know why we need this BuildContext, they'll naturally look to where it's used, and that tells them the same information that's in the comment.